charts/linkerd2/README.md

```diff
@@ -154,6 +155,7 @@ The following table lists the configurable parameters of the Linkerd2 chart and
 | `tap.keyPEM`                          | Certificate key for Tap component. If not provided then Helm will generate one.                                                                                                                                             ||
 | `webhookFailurePolicy`                | Failure policy for the proxy injector                                                                                                                                                 | `Ignore`                             |
 | `webImage`                            | Docker image for the web container                                                                                                                                                    | `gcr.io/linkerd-io/web`              |
+| `enforcedHostRegexp`                  | Host header validation regex for the dashboard. See the [Linkerd documentation](https://linkerd.io/2/tasks/exposing-dashboard) for more information                                   | `gcr.io/linkerd-io/web`              |
```

Comment:
The actual default value for this one is the empty string, which allows requests for `localhost`, `127.0.0.1`, `linkerd-web.linkerd.svc.cluster.local` and `linkerd-web.linkerd.svc` (in the latter two, the system will check for the actual namespace and cluster domain configured for this installation).
It would be great you could convey these details in the description of this entry :wink:

Reply:
Thanks for the catch. I updated the default value to an empty string.